internal/common/helpers.go

```diff
@@ -23,3 +26,13 @@ func PanicOnError(err error) {
 		panic(err)
 	}
 }
+
+// IsStringInSlice returns true if the string is present, false if not
+// slice must be sorted
+func IsStringInSlice(slice []string, s string) bool {
```

Comment:
oh, this actually always required a sorted slice. While doing refactorings, can you rename the method to `IsStringInSortedSlice` so it's obvious what it does? I think this is a big trap for someone who doesn't read comments.

Reply:
Good idea!